component/src/main/java/org/wso2/extension/siddhi/store/influxdb/InfluxDBEventTable.java

```diff
@@ -0,0 +1,733 @@
+                Object myObject = entry.getValue();
+                if (myObject instanceof String) {
+                    condition = condition.replaceFirst(Pattern.quote("*"), myObject.toString());
+                } else {
+                    condition = condition.replaceFirst(Pattern.quote("'*'"), myObject.toString());
+                }
+            }
+            findCondition = line.toString();
+            findCondition = findCondition + condition;
+        }
+        influxdb.setDatabase(database);
+        query = new Query(findCondition, database);
+        influxdb.query(query);
+        queryResult = influxdb.query(query);
+        return queryResult;
+    }
+
+    public void writePoints(List<Object[]> list) {
+
+        for (Object[] record : list) {
+            Map<String, String> insertTagMap = InfluxDBTableUtils.mapTagValuesToAttributes(record,
+                    this.attributeNames, this.tagPositions);
+            Map<String, Object> insertFieldMap = InfluxDBTableUtils.mapFieldValuesToAttributes(record,
+                    this.attributeNames, this.tagPositions);
+            long times = InfluxDBTableUtils.mapTimeToAttributeValue(record, this.tableName,
+                    this.timePosition);
+            influxdb.setDatabase(database);
+            BatchPoints batchPoints = BatchPoints.database(database).build();
+            Point point = Point.measurement(tableName)
+                    .time(times, TimeUnit.MILLISECONDS)
+                    .tag(insertTagMap)
+                    .fields(insertFieldMap)
+                    .build();
+            batchPoints.point(point);
+            influxdb.write(batchPoints);
+        }
+    }
+
+    public boolean checkDatabaseExists(String dbName) {
+
+        boolean isExists = false;
+        int index = 0;
+        Query query = new Query("SHOW DATABASES", database);
+        QueryResult queryResult = this.influxdb.query(query);
+        List<List<Object>> databaseList = queryResult.getResults()
+                .get(0)
+                .getSeries()
+                .get(0)
+                .getValues();
+        while (!isExists && index < databaseList.size()) {
+            if (databaseList.get(index).get(0).equals(dbName)) {
+                isExists = true;
+            }
+            index++;
+        }
+        return isExists;
+    }
+
+    @Override
+    protected RecordIterator<Object[]> query(Map<String, Object> parameterMap, CompiledCondition compiledCondition,
+                                             CompiledSelection compiledSelection, Attribute[] outputAttributes)
+            throws ConnectionUnavailableException {
+
+        InfluxDBCompiledSelection influxDBCompiledSelection = (InfluxDBCompiledSelection) compiledSelection;
+        InfluxDBCompiledCondition influxDBCompiledCondition = (InfluxDBCompiledCondition) compiledCondition;
+        Map<Integer, Object> consMap = influxDBCompiledCondition.getParametersConstants();
+        Query query1;
```

Comment:
Please avoid using 1, 2...'s in variable names unless the life time of the variable is very small.